test/vfd_swmr_bigset_writer.c

```diff
@@ -147,128 +198,150 @@ state_initializer(void)
                      .filetype         = H5T_NATIVE_UINT32,
                      .one_by_one_sid   = H5I_INVALID_HID,
                      .quadrant_dcpl    = H5I_INVALID_HID,
+                     .depth            = DEPTH,
                      .rows             = ROWS,
                      .cols             = COLS,
                      .ndatasets        = 5,
                      .asteps           = 10,
                      .nsteps           = 100,
+                     .part_chunk       = 0,
+                     .skip_chunk       = SKIP_CHUNK,
+                     .over_extend      = 1,
                      .filename         = {"", "", "", ""},
-                     .two_dee          = false,
-                     .wait_for_signal  = true,
+                     .expand_2d        = false,
+                     .test_3d          = false,
                      .vds              = vds_off,
                      .use_vfd_swmr     = true,
+                     .use_named_pipe   = true,
+                     .do_perf          = false,
+                     .cross_chunk_read = false,
                      .writer           = true,
                      .fixed_array      = false,
                      .one_dee_max_dims = {ROWS, H5S_UNLIMITED},
                      .chunk_dims       = {ROWS, COLS},
-                     .update_interval =
-                         (struct timespec){.tv_sec = 0, .tv_nsec = 1000000000UL / 30 /* 1/30 second */}};
+                     .fsp_size         = FSP_SIZE,
+                     .page_buf_size    = PAGE_BUF_SIZE,
+                     .tick_len         = TICK_LEN,
+                     .max_lag          = MAX_LAG,
+                     .flush_raw_data   = false,
```

Comment:
Looks like flush_raw_data is default to be false.  Does the tests in testvfdswmr.sh.in just test with the default flush_raw_data value?  If so, does the raw data verification all ok?

Reply:
I added a test case for flushing raw data in the test script.

Reply:
When flushing of raw data is disabled, I think the verification of raw data should fail?